bundle/edu.gemini.ags/src/main/java/edu/gemini/ags/gems/GemsGuideStarSearchOptions.java

```diff
@@ -144,48 +101,72 @@ public GemsInstrument getInstrument() {
      * @param nirBand      optional NIR magnitude band (default is H)
      * @return all relevant CatalogSearchCriterion instances
      */
-    public List<GemsCatalogSearchCriterion> searchCriteria(final ObsContext obsContext, final scala.Option<MagnitudeBand> nirBand) {
-        switch(tipTiltMode) {
-            case canopus: return Arrays.asList(
-                    canopusCriterion(obsContext, GemsGuideStarType.tiptilt),
-                    instrumentCriterion(obsContext, GemsGuideStarType.flexure, nirBand));
-            case instrument: return Arrays.asList(
-                    instrumentCriterion(obsContext, GemsGuideStarType.tiptilt, nirBand),
-                    canopusCriterion(obsContext, GemsGuideStarType.flexure));
-            default:
-            case both: return Arrays.asList(
-                    canopusCriterion(obsContext, GemsGuideStarType.tiptilt),
-                    instrumentCriterion(obsContext, GemsGuideStarType.flexure, nirBand),
-                    instrumentCriterion(obsContext, GemsGuideStarType.tiptilt, nirBand),
-                    canopusCriterion(obsContext, GemsGuideStarType.flexure)
-            );
-        }
+    public List<GemsCatalogSearchCriterion> searchCriteria(
+        final ObsContext                  obsContext,
+        final scala.Option<MagnitudeBand> nirBand
+    ) {
+        return Arrays.asList(
+                    canopusCriterion(obsContext),
+                    instrumentCriterion(obsContext, nirBand));
```

Comment:
So here usually there would be no `instrumentCriterion` (but it will depend on the flexure search option).